src/rpc/handlers/Ledger.cpp

```diff
@@ -45,6 +45,13 @@ doLedger(Context const& context)
         diff = params.at("diff").as_bool();
     }
 
+    if (params.contains(JS(full)) && params.contains(JS(accounts)))
+    {
+        if ((params.at(JS(full)).as_bool() &&
+             params.at(JS(accounts)).as_bool()) == false)
+            return Status{Error::rpcNO_PERMISSION, "noPermission"};
```

Comment:
We can use `rpcNOT_SUPPORTED` here or once we merge #396 we will be able to add and return a custom `clio` error for this if desired.